GitDiffMargin/source.extension.vsixmanifest

```diff
@@ -13,10 +13,11 @@
     <!--<Tags></Tags>-->
   </Metadata>
   <Installation>
-    <InstallationTarget Version="[11.0,15.0)" Id="Microsoft.VisualStudio.Pro" />
+    <InstallationTarget Version="[11.0,16.0)" Id="Microsoft.VisualStudio.Community" />

```

Comment:
Pro and Enterprise are implied. @madskristensen now uses a single installation target for his extensions as well. I made this change based on what I saw there.